nixos/tests/etcd.nix

```diff
@@ -10,6 +10,7 @@ import ./make-test-python.nix ({ pkgs, ... } : {
   nodes = {
     node = { ... }: {
       services.etcd.enable = true;
+      services.etcd.package = pkgs.etcd_3_4;
```

Comment:
Rather than changing it here the top level default should be changed.

```diff
diff --git a/pkgs/top-level/all-packages.nix b/pkgs/top-level/all-packages.nix
index 2cfaab8fe07..9f871cfcc0b 100644
--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -20935,7 +20935,7 @@ with pkgs;
     gn = gn1924;
   };
 
-  etcd = etcd_3_3;
+  etcd = etcd_3_4;
   etcd_3_3 = callPackage ../servers/etcd/3.3.nix { };
   etcd_3_4 = callPackage ../servers/etcd/3.4.nix { };
 
```